src/langapi/Makefile

```diff
@@ -1,4 +1,4 @@
-SRC = mode.cpp language_ui.cpp languages.cpp language_util.cpp
```

Comment:
The compiler needs some place to actually store the virtual tables. With header-only files that induce vtables there would be duplication, and the linker may or may not be able to resolve that. Thus: please keep that file, even when seemingly more complex to manage. (Use inlining for performance, not for laziness. - A rule the code base doesn't quite follow, though.)